auter.yumdnfModule

```diff
@@ -164,9 +169,13 @@ function apply_updates() {
     # We don't want to allow the user to interrupt a yum/dnf transaction or Bad Things Happen.
     trap '' SIGINT SIGTERM
     rotate_file "${DATADIR}/last-apply-output-${CONFIGSET}"
-    APPLYOUTPUT+=$("${PACKAGE_MANAGER}" "${UPDATEACTION}" -y "${PACKAGEMANAGEROPTIONS[@]}" "${RPMS[@]}" 2>&1)
+    if APPLYOUTPUT+=$("${PACKAGE_MANAGER}" "${UPDATEACTION}" -y "${PACKAGEMANAGEROPTIONS[@]}" "${RPMS[@]}" 2>&1); then
+      APPLYOUTPUT+="\\nSTATUS:SUCCESS:Package updates applied"
+    else
+      APPLYOUTPUT+="\\nSTATUS:FAILED:Package updates failed"
+    fi
     default_signal_handling
-    echo -e "$APPLYOUTPUT" &>"${DATADIR}/last-apply-output-${CONFIGSET}"
+    echo -e "$APPLYOUTPUT" > "${DATADIR}/last-apply-output-${CONFIGSET}"
```

Comment:
Wondering why the change from `&>` to `>` ?

Reply:
I couldn't work out why we would need stderr from an echo. Have I missed something?

Reply:
Good point!
I was thinking it was from the output of the commands themselves! 
Approved then!! :)

Reply:
Awesome, thanks for pulling it up! I'll look at getting it merged and into a release.